test/NuGet.Core.Tests/NuGet.Configuration.Test/SettingsTests.cs

```diff
@@ -2043,35 +2043,80 @@ public void LoadSettings_AddsV3ToEmptyConfigFile_OnlyFirstTime()
-                                    configFileName: null,
-                                    machineWideSettings: null,
-                                    loadUserWideSettings: true,
-                                    useTestingGlobalPath: true);
+                var settings = Settings.LoadSettings(
+                    root: mockBaseDirectory,
+                    configFileName: null,
+                    machineWideSettings: null,
+                    loadUserWideSettings: true,
+                    useTestingGlobalPath: true);
+
                 // Assert
-                text = SettingsTestUtils.RemoveWhitespace(File.ReadAllText(Path.Combine(mockBaseDirectory, "TestingGlobalPath", "NuGet.Config")));
-                result = SettingsTestUtils.RemoveWhitespace(@"<?xml version=""1.0"" encoding=""utf-8""?>
+                var text = SettingsTestUtils.RemoveWhitespace(File.ReadAllText(Path.Combine(mockBaseDirectory, "TestingGlobalPath", "NuGet.Config")));
+                var v3feed = SettingsTestUtils.RemoveWhitespace(@"<add key=""nuget.org"" value=""https://api.nuget.org/v3/index.json"" protocolVersion=""3"" />");
+                var result = SettingsTestUtils.RemoveWhitespace(@"<?xml version=""1.0"" encoding=""utf-8""?>
 <configuration>
 </configuration>");
 
+                text.Should().NotContain(v3feed);
+                text.Should().Be(result);
+            }
+        }
+
+        [Fact]
+        public void LoadSettings_NonExistingUserWideConfigFile_CreateUserWideConfigFileAndAddsV3Feed()
+        {
+            using (var mockBaseDirectory = TestDirectory.Create())
+            {
+                // Arrange
+                var nugetConfigPath = Path.Combine(mockBaseDirectory, "TestingGlobalPath", "NuGet.Config");
+                File.Exists(nugetConfigPath).Should().BeFalse();
+
+                // Act
+                var settings = Settings.LoadSettings(
+                    root: mockBaseDirectory,
+                    configFileName: null,
+                    machineWideSettings: null,
+                    loadUserWideSettings: true,
+                    useTestingGlobalPath: true);
+
+                // Assert
+                File.Exists(nugetConfigPath).Should().BeTrue();
+                var text = SettingsTestUtils.RemoveWhitespace(File.ReadAllText(Path.Combine(mockBaseDirectory, "TestingGlobalPath", "NuGet.Config")));
+                var v3feed = SettingsTestUtils.RemoveWhitespace(@"<add key=""nuget.org"" value=""https://api.nuget.org/v3/index.json"" protocolVersion=""3"" />");
+                var result = SettingsTestUtils.RemoveWhitespace(@"<?xml version=""1.0"" encoding=""utf-8""?>
+<configuration>
+    <packageSources>
+        <add key=""nuget.org"" value=""https://api.nuget.org/v3/index.json"" protocolVersion=""3"" />
+    </packageSources>
+</configuration>");
+
+                text.Should().Contain(v3feed);
                 text.Should().Be(result);
```

Comment:
Like the other tests, asserting that `text.Should.Be(result)`, where `result` already contains the nuget.org feed, makes `text.Should().Contain(v3feed)` redundant.

Reply:
Thanks. Fixed.